scripts/steps/build-api.js

```diff
@@ -110,6 +104,105 @@ const UNWRAP_UTILITIES = new Set([
   'Readonly'
 ])
 
+const EXCLUDED_SUBMENU_KINDS = new Set(['Type alias', 'Interface', 'Namespace'])
+const EXCLUDED_TREE_KINDS = new Set(['Namespace'])
+const CORE_ENTITIES = new Set([
+  'BaseNode',
+  'ClientNode',
+  'Connection',
+  'Log',
+  'LogStore',
```

Comment:
`LogStore` is also to `Others` since creating custom log stores is hard and rare